style.css

```diff
@@ -0,0 +1,378 @@
+  width: 37.5%;
+  height: 9vh;
+  border: none;
+  color: white;
+  font-weight: 700;
+  font-size: 1em;
+  font-family: 'Inter', sans-serif;
+}
+
+.grid3 {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: auto;
+  flex-wrap: wrap;
+  flex-direction: column;
+  text-align: left;
+  background-image: url("./Normal_Button/project_background.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: white;
+  margin-bottom: 3vh;
+}
+
+.grid3 h2 {
+  padding-top: 9vh;
+  font-size: 2em;
+  padding-left: 3vh;
+  margin-bottom: 3vh;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+}
+
+.grid3 ul {
+  display: flex;
+  margin-left: 1vh;
+}
+
+.grid3 ul p {
+  font-size: 1em;
+  padding: 2vh;
+  background: rgba(255, 255, 255, 0.24);
+  margin-left: 2vh;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+}
+
+.see-project2 {
+  width: 100%;
+  background-color: #ff6b00;
+  border: none;
+  height: 9vh;
+  color: white;
+  font-size: 1.2em;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+}
+
+.about-section {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 85%;
+  grid-template-rows: repeat(2, auto);
+  justify-content: center;
+  background-image: url("./Normal_Button/about-me-back.png");
+  background-repeat: no-repeat;
+  background-position: 100% 21%;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  height: 115vh;
+  background-image: url("./Normal_Button/Vector-about.png");
+  background-repeat: no-repeat;
+  background-position: bottom;
+}
+
+.about-me h2 {
+  font-size: 4.5em;
+  text-align: center;
+  color: #ff6b00;
+  margin-bottom: 3vh;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+}
+
+.get-resume {
+  width: 28vh;
+  height: 9vh;
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+  font-size: 1em;
+  font-weight: 700;
+  margin-top: 3vh;
+  font-family: 'Inter', sans-serif;
+}
+
+.knowledge {
+  margin-top: 20vh;
+  display: flex;
+  flex-direction: column;
+  color: white;
+  text-align: center;
+}
+
+.knowledge li {
+  font-size: 1.5em;
+  font-weight: 400;
+  padding: 3vh;
+  font-family: 'Inter', sans-serif;
+}
+
+.knowledge h2 {
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+}
+
+button:hover {
+  cursor: pointer;
+  border: #f55800 solid 1px;
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+button:active {
+  border: none;
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
+@media (min-width: 375px) {
+  .dropdown-menu {
+    display: none;
+  }
+
+  .headline h1 {
+    font-family: 'Crete Round', serif;
+    font-size: 3em;
+    color: #ff6b00;
+  }
+
+  .headline h2 {
+    font-family: 'Roboto', sans-serif;
+    font-size: 1.5em;
+    color: white;
+  }
+
+  .headline p {
+    color: white;
+    font-size: 1.2em;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .social-media {
+    width: 30vh;
+    justify-content: space-around;
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
Good use of the media queries! 
However, since you are only working on the mobile version of your portfolio maybe it's best if you temporarily remove it and merge all of its content into the main code. That way you will have a working mobile version which you can change later on into a desktop version! what do you think? 🤔